src/api/common/utils/EntityUtils.ts

```diff
@@ -47,16 +56,22 @@ export const POST_MULTIPLE_LIMIT = 100
  * Tests if one id is bigger than another.
  * @param firstId The element id that is tested if it is bigger.
  * @param secondId The element id that is tested against.
+ * @param typeModel the type the Ids belong to
  * @return True if firstId is bigger than secondId, false otherwise.
  */
-export function firstBiggerThanSecond(firstId: Id, secondId: Id): boolean {
-	// if the number of digits is bigger, then the id is bigger, otherwise we can use the lexicographical comparison
-	if (firstId.length > secondId.length) {
-		return true
-	} else if (secondId.length > firstId.length) {
-		return false
+export function firstBiggerThanSecond(firstId: Id, secondId: Id, typeModel?: TypeModel): boolean {
```

Comment:
Seems a bit risky but maybe okay

Reply:
added the note about usage contract from the previous impl to the doc comment.

Reply:
added the note about usage contract from the previous impl to the doc comment.